tests/tt_metal/tt_metal/unit_tests_fast_dispatch/command_queue/test_worker_config_buffer.cpp

```diff
@@ -0,0 +1,39 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <memory>
+
+#include "gtest/gtest.h"
+#include "tt_metal/impl/dispatch/worker_config_buffer.hpp"
+
+using std::vector;
+using namespace tt::tt_metal;
+
+namespace working_config_buffer_tests {
+
+TEST(WorkingConfigBuffer, MarkCompletelyFull) {
+    WorkerConfigBufferMgr mgr;
+    mgr.init_add_core(1024, 1024);
+    mgr.init_add_core(2, 1024);
+
+    auto reservation = mgr.reserve({12, 12});
+    mgr.alloc(1);
+
+    mgr.mark_completely_full(5);
+
+    // Allocation would succed, except buffer is marked completely full.
```

Comment:
```suggestion
    // Allocation would suceed, except buffer is marked completely full.
```

Reply:
Just noticed I made a typo in my correction as well 😞 . Should have been `succeed`